collectors/o365/o365_collector.js

```diff
@@ -0,0 +1,155 @@
+/* -----------------------------------------------------------------------------
+ * @copyright (C) 2019, Alert Logic, Inc
+ * @doc
+ *
+ * o365 class.
+ *
+ * @end
+ * -----------------------------------------------------------------------------
+ */
+'use strict';
+
+const moment = require('moment');
+const PawsCollector = require('@alertlogic/paws-collector').PawsCollector;
+const m_o365mgmnt = require('./lib/o365_mgmnt');
+
+const parse = require('@alertlogic/al-collector-js').Parse;
+
+
+const typeIdPaths = [
+   { path: ['CreationTime'] }
+];
+
+const tsPaths = [
+    { path: ['RecordType'] }
+];
+
+console.log(process.env.O365_CONTENT_STREAMS)
+class O365Collector extends PawsCollector {
+    constructor(context, creds) {
+        super(context, creds, 'o365');
+    }
+    
+    pawsInitCollectionState(event, callback) {
+        let startTs = process.env.paws_collection_start_ts ? 
+                process.env.paws_collection_start_ts :
+                    moment().toISOString();
+        let endTs;
+
+        if(moment().diff(startTs, 'days') > 7){
+            // Subtracting less than 7 days to avoid weird race conditions with the azure api...
+            // Missing about 9 seconds of historical logs shouldn't be too bad.
+            startTs = moment().subtract(6.9999, 'days').toISOString();
+            console.log("Start timestamp is more than 7 days in the past. This is not allowed in the MS managment API. setting the start time to 7 days in the past");
+        }
+        
+        if(moment().diff(startTs, 'hours') > 24){
+            endTs = moment(startTs).add(24, 'hours').toISOString();
```

Comment:
worth console logging something out here, in the same way it's done above for 7 days?